gui/__tests__/selectors.test.js

```diff
@@ -125,6 +126,30 @@ describe('selectors', () => {
     })
   })
 
+  describe('jobRunsCountSelector', () => {
+    it('returns the number of runs for the job', () => {
+      const state = {
+        jobs: {
+          items: {
+            jobA: {id: 'jobA', runs: ['runA', 'runB', 'runC', 'runD', 'runE', 'runF']}
+          }
+        },
+        jobRuns: {
+          items: {
+            'runA': {id: 'runA'},
```

Comment:
Do we need A through F? Wouldn't A + B be sufficient?

Reply:
It shows that the count can be > 5 which is the limit of `latestJobRuns`

Reply:
How do you feel about updating the test name so it's something like:

```javascript
it('returns the run total for the job, not just the total of latest runs', () => {
   // ...
})
```

Reply:
Could it be configured to paginate at 1?

Reply:
Currently there's no ability to paginate that relationship

Reply:
Right, I'm just suggesting we inject the value `5` and set it to something like `1` for testing. This way we don't have to rewrite the tests if we decide to change it, and the tests can be shorter.